src/lib/mina_commands/mina_commands.ml

```diff
@@ -97,7 +97,9 @@ let setup_and_submit_user_command t (user_command_input : User_command_input.t)
                 | `Not_broadcasted ->
                     "not_broadcasted" ) )
           ; ( "valid_commands"
-            , `List (List.map ~f:User_command.to_yojson valid_commands) )
+            , `List
```

Comment:
Log format change for `Send_user_commands` RPC. It's only logging on server, output format on the user side is not affected.